zebra-network/src/constants.rs

```diff
@@ -21,6 +21,29 @@ use zebra_chain::{
     serialization::Duration32,
 };
 
+/// The name of the current Zebra release.
+//
+// TODO: generate this from crate metadata (#2375)
+pub const RELEASE_NAME: &str = "Zebra:1.0.0-rc.6";
+
+/// The date of the current Zebra release.
+///
+/// This must be a valid `chrono::DateTime` with timezone string.
+pub const RELEASE_DATE: &str = "2023-03-23 00:00:00 +00:00";
+
+/// The maximum number of days after `RELEASE_DATE` where a Zebra server runs.
+///
+/// Notes:
+///
+/// - Zebra will exit with a panic if the current date is bigger than the `RELEASE_DATE` date plus this number of days.
+pub const EOS_PANIC_AFTER: u64 = 180;
```

Comment:
Let's ask the team how long we should support Zebra?
Currently `zcashd` uses 16 weeks.

Reply:
Unless anyone sees any strong reason why we should do it differently I would match what `zcashd` does. I also wouldn't block on this necessarily, as long as the length of time can be easily changed as needs evolve. I feel like the mechanism is more important than the actual choice of support time length.

Reply:
> Unless anyone sees any strong reason why we should do it differently I would match what zcashd does.

I don't have a strong opinion here but I think Str4d mentioned zcashd's 16 weeks feeling too short in hindsight.

Reply:
Since we might be fixing some serious network security issues as part of the audit, I'd like to make the support lifetime as short as possible for now. That way, all vulnerable versions of Zebra will be removed from the network sooner.

For example, if we merge this PR with a 6 month limit, we'll have those Zebra versions on the network until November 2023. But if we use a 16 week limit, they'll be around until August 2023.

We can extend it later once we're confident Zebra is secure and working well.

Reply:
I changed to the exact 16 weeks(112 days) at https://github.com/ZcashFoundation/zebra/pull/6351/commits/11f34721b73b73b919545c5a45e80c7004fa1d58